### ISSUE TITLE
[Bug]: Pages Module "Expand All" Does Not Function

### ISSUE THREAD

**Issue by mitchelsellers** [dnnsoftware/Dnn.Platform#5695]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

When in the Pages view the [Expand All] button in the upper left corner when clicked does not do anything.

No error messages are shown.

![image](https://github.com/dnnsoftware/Dnn.Platform/assets/5659113/86aa50fa-806f-4db1-9ad9-8df4678fb3ee)

### Steps to reproduce?

1. Open the Persona bar
2. Select `Pages`
3. Try to click on [EXPAND ALL]

### Current Behavior

Nothing happens

### Expected Behavior

All tree nodes should be expanded

### Relevant log output

```shell
No visible logs were found on the console, Event Log, or Log4Net Logs
```

### Affected Versions

9.11.2 (latest release)

### What browsers are you seeing the problem on?

Firefox, Chrome, Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by mitchelsellers** [comment]
@valadas Can you confirm that you see this behavior as well?

**Comment by valadas** [comment]
Yes, I confirm. now odd thing is if you expand something then it changes to "Collapse All" and after that both "Expand All" and "Collapse All" start working fine...

My guess is that we need to have some child nodes loaded up before this feature works...
